src/Search/SearchTypeSwitcher/SearchTypeSwitcher.jsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames/bind';
+import React from 'react';
+import styles from './SearchTypeSwitcher.scss';
+
+function SearchTypeSwitcher() {
```

Comment:
I would like switcher to switch search type and change state of the Search component